### DISCUSSION TITLE
Is there a UI designer app used for development?

### DISCUSSION THREAD

**Discussion by Oxalin** [buttercup/buttercup-desktop#1252] [Q&A] [answered]
I was wondering if Buttercup Desktop was being developed with a UI designer? I know of React Studio, but its only for macOS.

**Comment by perry-mitchell** [comment, marked as answer]
We're using BlueprintJS: https://blueprintjs.com

Sallar, one of the original founders, was definitely far better at design work than I am, but he's since left the project.

We're considering a system wide design change in the future but it's not anywhere near a top priority just yet.
